public_html/templates/index.html

```diff
@@ -71,7 +71,7 @@
 					<div class="row-div col-sm-3 text-center wikiproject-div">
 						{% if row.wikiprojects is not empty %}
 							{% for wp in row.wikiprojects %}
-								<span class="row-div wproject">{{ wp }}</span>
+								<a class="row-div wproject" href="#">{{ wp }}</a>
```

Comment:
Well I figure we'd rotate between the 10, starting from the top of the page. I guess there's the complication that if they select a WikiProject via search, it's colour would need to match that of the same WikiProject bubble on the page. But yes, this effort is totally unnecessary and not worth our while =P